website/pages/en/network/developing.mdx

```diff
@@ -2,52 +2,122 @@
 
-In order to make queries, developers must generate an API key, which can be done in Subgraph Studio. This API key must be funded with GRT, in order to pay query fees. Developers can set a maximum query fee, in order to control their costs, and limit their API key to a given subgraph or origin domain. Subgraph Studio provides developers with data on their API key usage over time.
+Learn the detailed specifics to [create a subgraph](/developing/creating-a-subgraph/).
 
-Developers are also able to express an Indexer preference to the gateway, for example preferring Indexers whose query response is faster, or whose data is most up to date. These controls are set in Subgraph Studio.
+### Subgraph Lifecycle
 
-### Updating Subgraphs
+Here is a general view of a subgraph’s cycle:
 
-After a time a subgraph developer may want to update their subgraph, perhaps fixing a bug or adding new functionality. The subgraph developer may deploy new version(s) of their subgraph to Subgraph Studio for rate-limited development and testing.
+![Subgraph Lifecycle](/img/subgraph-lifecycle.png)
 
-Once the Subgraph Developer is ready to update, they can initiate a transaction to point their subgraph at the new version. Updating the subgraph migrates any signal to the new version (assuming the user who applied the signal selected "auto-migrate"), which also incurs a migration tax. This signal migration should prompt Indexers to start indexing the new version of the subgraph, so it should soon become available for querying.
+### Subgraph Development
 
-### Deprecating Subgraphs
+Subgraph development starts with local development and staging.
 
-At some point a developer may decide that they no longer need a published subgraph. At that point they may deprecate the subgraph, which returns any signalled GRT to the Curators.
+- You can use the same local setup whether you're building for The Graph Network or a local Graph Node.
+  - You can leverage `graph-cli` and `graph-ts` to build your subgraph.
+- You are encouraged to use tools such as [Matchstick](https://github.com/LimeChain/matchstick) for unit testing to improve the robustness of your subgraphs.
 
-### Diverse Developer Roles
+> Only subgraphs on [supported networks](/developing/supported-networks/) will earn indexing rewards. Subgraphs that fetch data from IPFS are also **not** eligible.
 
-Some developers will engage with the full subgraph lifecycle on the network, publishing, querying and iterating on their own subgraphs. Some may be focused on subgraph development, building open APIs which others can build on. Some may be application focused, querying subgraphs deployed by others.
+## Deploy to Subgraph Studio
 
-### Developers and Network Economics
+Once defined, a subgraph can be built and deployed to [Subgraph Studio](/deploying-a-subgraph-to-studio/). In Subgraph Studio, you can do the following:
```

Comment:
Is the subgraph not "built" at this point? (By build we mean "developed', which was the previous step, right?)